cucumber/features/careerPage.feature

```diff
@@ -2,3 +2,30 @@ Feature: EPAM job searching
   As a Job searcher
   I want to browser through EPAM Job offers by various criteria
   So I can find to best fitting offer for me
+
+Scenario Outline: Search for <PositionName> in <City>
+  Given the career page is opened
+  Then the cookie bar should be closed
+  And the logo should be visible
+  And the searchform should be visible
+  
+  When <Country> and <City> selected in the location filter box
+  Then the <City> location should be selected
+
+  When <Department> selected in the department filter box
+  Then the <Department> department should be selected
+
+  When the search button is clicked
+  Then should have a proper job found for <PositionName> position
+  And the proper location in the first result should be <Country>
+  And description should be visible in the first result
+  And apply button should be visible for <PositionName> position
```

Comment:
The apply button is in the first result item. Could you please add a "1st" number to this step?

Reply:
done